core/src/main/resources/hudson/util/HudsonIsRestarting/index.jelly

```diff
@@ -56,6 +56,12 @@ THE SOFTWARE.
                         <p class="restarting">
                             ${%Your browser will reload automatically when Jenkins is ready.}
                         </p>
+                        <j:if test="${it.safeRestart}">
+                            <p class="safe-restarting">
+                                <b>${%Safe Restart}</b><br/> ${%Jobs on agents can usually continue.}
```

Comment:
Use CSS for presentation, avoid br except for things like addresses.

strong is probably more semantically correct in this case than the bold tag

Reply:
Done :)
![image](https://user-images.githubusercontent.com/41284403/216296181-8ddbac24-a3a0-405c-a7e2-7814ed1038d5.png)